tests/test_eligibility.py

```diff
@@ -83,12 +83,12 @@ def _do_test(country, council_tax_band, epc_rating):
     page = form.submit().follow()
 
     form = page.get_form()
-    form["rrn"] = "2222-2222-2222-2222-2222"
+    form["lmk"] = "222222222222222222222222222222222"
```

Comment:
for the most part test changes here are only to the selecting EPC part of tests

anything beyond that (saving response etc) is left to be fixed in PC-1297 #361 

tests are not expected to be passing in this PR